modules/core/test/v2/unit/wallets.ts

```diff
@@ -94,6 +98,22 @@ describe('V2 Wallets:', function() {
       .reply(200, {});
       yield wallets.add({ label: 'label', enterprise: 'enterprise', type: 'custodial' });
     }));
+
+    it('creates a wallet with custom gasPrice', co(function *createWalletCustomGasPrice() {
+      const ethBitGo = new TestBitGo({ env: 'mock' });
+      ethBitGo.initializeTestVars();
+      const ethWallets = ethBitGo.coin('teth').wallets();
+      nock(bgUrl)
+        .post('/api/v2/teth/wallet', function(body) {
+          body.type.should.equal('custodial');
+          body.should.not.have.property('keys');
+          body.should.not.have.property('m');
+          body.should.not.have.property('n');
+          return true;
```

Comment:
These tests are intended to check that your custom parameters are being passed correctly. To do so, add a check in this callback to ensure that the custom gas price is passed as intended

Reply:
There's actually a slightly cleaner way to do this - set up your nock so it only matches if the body params are as you expect.

This isn't incorrect though I'm so not requesting changes for it.